src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java

```diff
@@ -56,7 +59,16 @@ public Path getCloudSdkPath() {
     return searchPaths(possiblePaths);
   }
 
-  private void getLocationsFromPath(List<String> possiblePaths) {
+  private static String getLocalAppDataLocation() {
```

Comment:
I'd add a comment here that the single-user installation option (the default) places the sdk in the LocalAppData